src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -399,9 +386,49 @@ public void reset() {
     logger.verbose("Config.setUserIdentifier(null);");
   }
 
-  private void trackVideo(String eventName, Properties properties) {
+  private void trackVideo(String eventName, TrackPayload track) {
+
+    if (heartbeatTrackingServer == null) {
+      logger.verbose(
+          "Please enter a Heartbeat Tracking Server URL in your Segment UI "
+              + "Settings in order to send video events to Adobe Analytics");
+      return;
+    }
+
     switch (eventName) {
-      case "Video Content Started":
+      case "Video Playback Started":
+        Context context = analytics.getApplication();
+        Properties properties = track.properties();
+        config = new MediaHeartbeatConfig();
+
+        config.trackingServer = heartbeatTrackingServer;
+        if (properties.get("channel") != null) {
+          config.channel = properties.getString("channel");
+        } else {
+          config.channel = "";
+        }
+        // default app version to 0.0 if not otherwise present b/c Adobe requires this value
+        if (!isNullOrEmpty(context.getPackageName())) {
```

Comment:
We could just do save this once during initialization

Reply:
You are totally right. Just fixed it.